src/EventSubscriber.php

```
@@ -0,0 +1,123 @@
+            } else {
+                $io->writeError('[scaffold] Non-string entry in allowed-packages ignored.');
+            }
+        }
+
+        return $allowed;
+    }
+
+    private function runScaffold(Event $event, bool $fullScaffold): void
+    {
+        $composer = $event->getComposer();
+        $io = $event->getIO();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $io. Configured minimum length is 3. (undefined)

(ShortVariable)